owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/files/RemoteFile.java

```diff
@@ -115,7 +111,7 @@ public RemoteFile(String path) {
     }
 
     public RemoteFile(final Response davResource, String userId) {
```

Comment:
Creating a RemoteUtil.kt class is not a bad idea but why haven't you converted `RemoteFile.java` to kotlin directly?

Reply:
Because it may require to perform also some changes to other files. We will migrate this RemoteFile to kotlin in next Synchronization epic for sure 👍